awx/main/analytics/collectors.py

```diff
@@ -237,29 +237,33 @@ def _copy_table(table, query, path):
-                                 main_unifiedjob.schedule_id, 
-                                 main_unifiedjob.execution_node, 
-                                 main_unifiedjob.controller_node, 
-                                 main_unifiedjob.cancel_flag, 
-                                 main_unifiedjob.status, 
-                                 main_unifiedjob.failed, 
-                                 main_unifiedjob.started, 
-                                 main_unifiedjob.finished, 
-                                 main_unifiedjob.elapsed, 
-                                 main_unifiedjob.job_explanation, 
-                                 main_unifiedjob.instance_group_id
-                                 FROM main_unifiedjob, django_content_type
-                                 WHERE main_unifiedjob.created > {} AND 
-                                 main_unifiedjob.polymorphic_ctype_id = django_content_type.id AND 
-                                 main_unifiedjob.launch_type != 'sync'
-                                 ORDER BY main_unifiedjob.id ASC) TO STDOUT WITH CSV HEADER'''.format(since.strftime("'%Y-%m-%d %H:%M:%S'"))    
+    unified_job_query = '''COPY (SELECT main_unifiedjob.id,
+                                main_unifiedjob.polymorphic_ctype_id,
+                                django_content_type.model,
+                                main_project.organization_id,
+                                main_organization.name as organization_name,
+                                main_unifiedjob.created,  
+                                main_unifiedjob.name,  
+                                main_unifiedjob.unified_job_template_id, 
+                                main_unifiedjob.launch_type, 
+                                main_unifiedjob.schedule_id, 
+                                main_unifiedjob.execution_node, 
+                                main_unifiedjob.controller_node, 
+                                main_unifiedjob.cancel_flag, 
+                                main_unifiedjob.status, 
+                                main_unifiedjob.failed, 
+                                main_unifiedjob.started, 
+                                main_unifiedjob.finished, 
+                                main_unifiedjob.elapsed, 
+                                main_unifiedjob.job_explanation, 
+                                main_unifiedjob.instance_group_id
+                                FROM main_unifiedjob, django_content_type, main_job, main_project, main_organization
+                                WHERE main_unifiedjob.created > {}
+                                AND main_project.unifiedjobtemplate_ptr_id = main_job.project_id
+                                AND main_unifiedjob.polymorphic_ctype_id = django_content_type.id
+                                AND main_unifiedjob.launch_type != 'sync'
+                                AND main_organization.id = main_project.organization_id
```

Comment:
@AlanCoding @rooftopcellist we're going to have to reconsider this after your merge of the "org on JT" feature.

Reply:
@ryanpetrello I have discussed this with @AlanCoding and we decided to merge this first and then make any necessary changes when Alan's PR lands.

Reply:
Perfect!